go.mod

```diff
@@ -123,12 +123,12 @@ require (
 	go.opentelemetry.io/otel/exporters/otlp/internal/retry v1.14.0 // indirect
 	go.opentelemetry.io/otel/metric v0.37.0 // indirect
 	go.opentelemetry.io/proto/otlp v0.19.0 // indirect
-	golang.org/x/crypto v0.1.0 // indirect
+	golang.org/x/crypto v0.11.0 // indirect
 	golang.org/x/mod v0.9.0 // indirect
-	golang.org/x/net v0.8.0 // indirect
+	golang.org/x/net v0.13.0 // indirect
```

Comment:
this moves the 1.7 branch ahead of main; do we need 0.13 or is 0.12 enough for the cve? (potentially backporting from main)

Otherwise if we really need v0.13 we should make sure main is updated first https://github.com/containerd/containerd/blob/e1655fe915a74be8315ffe8c3f8c2469009b4498/go.mod#L127

Reply:
:heavy_plus_sign: If we want 0.13.0, let's update main first.

Reply:
Sure. I created https://github.com/containerd/containerd/pull/9184